### ISSUE TITLE
Anybody have a visitor pattern example?

### ISSUE THREAD

**Issue by mrunix** [rrevenantt/antlr4rust#11]
Thank you very much!

**Comment by fruffy** [comment]
Yes. I tried this tool out with the openCypher ANTLR grammar and it works nicely out of the box. Thanks a lot for this, a very useful tool. However, the visitor API is somewhat unclear. There is one small example in the tests directory, but it is difficult to understand how the visitor can be used. Where is the core API defined?

**Comment by newca12** [comment]
https://github.com/newca12/playground-rs/tree/main/antlr-calc

**Comment by yunusemredilber** [comment]
https://github.com/yunusemredilber/antlr_rust_language
